openproblems/tasks/_batch_integration/batch_integration_graph/methods/combat.py

```diff
@@ -5,10 +5,10 @@
 
 @method(
     method_name="Combat",
-    paper_name="Sc",
-    paper_url="temp",
-    paper_year=2020,
-    code_url="",
+    paper_name="A test metric for assessing single-cell RNA-seq batch correction",
+    paper_url="https://www.nature.com/articles/s41592-018-0254-1",
+    paper_year=2018,
+    code_url="github.com/theislab/scanpy",
```

Comment:
paper info is from kBet and not ComBat... I believe combat is johnson & rabinovic from about 2007.